docs/auto-generate-skus/README.md

```diff
@@ -28,12 +28,14 @@ shopify/products/update
 
 ## Documentation
 
-Save time by letting this task keep your variant SKUs up to date, generating them based on your product handle, and the initials of each variant option. Optionally, choose to have the task skip updating variants that already have SKUs. Useful for large catalogs, or for anyone who has a consistent SKU format. Like all tasks, this one is easy to update for your business's needs - get in touch if you need a hand.
+Save time by letting this task keep your variant SKUs up to date, generating them based on your product handle, and the initials of each variant option. Optionally, choose to have the task skip updating variants that already have SKUs. Useful for large catalogs, or for anyone who has a consistent SKU format.
+
+Please note: This task updates SKUs for all products, whether or not they're configured with options and variants.
```

Comment:
This line was originally added to the very beginning of the task docs, as a new paragraph (in 40b884658a8d4ca3aa62eb7067d91d332a37d310). This wasn't a good place for it; the first paragraph is used as the task summary, used in usemechanic.com and in the in-app task search. So, I've moved it down here, and I've rephrased it a bit (in 7c74ffc67b9c0b875bd63b2f23dbec643e33a4bb).